documentation/smg/08_setting_up_jenkins_ci.md

```diff
@@ -325,3 +325,49 @@ withCredentials([string(credentialsId: 'TOKEN_ID', variable: 'secret')]) {
   """
 }
 ```
+
+## Established Jenkins pipelines
+
+### PR
+
+#### Herbert
```

Comment:
It would be good to add one figure showing your convention for the symbols in the diagrams. Even if it is a standard, it will be helpful to have the convention on-hand so that readers can immediately appreciate what you are saying.